src/sidebar.js

```diff
@@ -200,9 +197,9 @@ class Sidebar extends React.Component {
   }
 
   render() {
-    let sidebarStyle = styles.sidebar,
+    let sidebarStyle = Object.assign({}, styles.sidebar, { zIndex: this.props.zIndex + 1 }),
```

Comment:
Ok, I'll change it to use `update()` later. Actually I was surprised you use this approach and mix es6 with es5 features. You're using babel to compile code anyway and it includes its polyfills. There are many places where the code could be written more es6 (7?) way.